src/fronius-api.ts

```diff
@@ -78,6 +78,7 @@ export interface Inverter {
   E_Total: number;
   E_Year: number;
   P: number;
+  SOC: number; // percentage load of Battery/Akku
```

Comment:
Sorry I didn't see this and your other PRs earlier.

For Fronius inverters that don't have a battery connected, there doesn't seem to be a `SOC` property in the response.

```json
{
    "DT": 115,
    "E_Day": 29692,
    "E_Total": 58437300,
    "E_Year": 8686043,
    "P": 0
}
```

So I think this needs to be an optional property type.

```suggestion
  SOC?: number; // percentage load of Battery/Akku
```